manifests/quick-start-minimal.yaml

```diff
@@ -615,6 +615,11 @@ data:
       requests:
         cpu: 10m
         memory: 64Mi
+  imageCommandIndex: |
+    argoproj/argosay:v1: [cowsay]
+    argoproj/argosay:v2: [/argosay]
+    docker/whalesay:latest: [cowsay]
+    python:alpine3.6: [python3]
```

Comment:
I don't think command is enough because you need to allow a user to define both command and args.